apps/judicial-system/web/src/routes/Judge/HearingArrangements/HearingArrangements.tsx

```diff
@@ -49,6 +49,9 @@ export const HearingArrangements: React.FC = () => {
   const [modalVisible, setModalVisible] = useState(false)
   const [workingCase, setWorkingCase] = useState<Case>(null)
   const [isStepIllegal, setIsStepIllegal] = useState<boolean>(true)
+  const [isSendingNotification, setIsSendingNotification] = useState<boolean>(
```

Comment:
ættum frekar að nota loading sem við fáum úr useMutation. 

https://www.apollographql.com/docs/react/data/mutations/#tracking-loading-and-error-states